project_multifamily_beta/housing_characteristics/Misc Pool Heater.tsv

```diff
@@ -1,2 +1,3 @@
-Option=Electric	Option=Gas	Option=None
-0.004000	0.014000	0.982000
+Dependency=Misc Pool	Option=Electric	Option=Gas	Option=None
+None	0.000000	0.000000	1.000000
+Has Pool	0.053333	0.186667	0.760000
```

Comment:
Where are these values from?

Reply:
I took these from the values for National Average in the options lookup.

* 7.5% of houses have pools (pool_pump_mult=0.075)
* 1.8% of houses have pool heaters (pool_heater_elec_mult=0.004 + pool_heater_gas_mult=0.014)

So 1.8/7.5=24% of homes with pools have pool heaters, and 100-24=76% of homes with pools do not have pool heaters.